rspamd.yaml

```
@@ -0,0 +1,104 @@
+package:
+  name: rspamd
+  version: "3.12.1"
+  epoch: 0
+  description: "Rapid spam filtering system"
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+      - glib
+      - libarchive
+      - libevent
+      - libicu77
+      - libmagic
+      - libsodium
+      - luajit
+      - openssl
+      - pcre2
+      - sqlite-libs
+      - zlib
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cmake
+      - curl-dev
+      - gcc-14-default
+      - glib-dev
+      - icu-dev
+      - libarchive-dev
+      - libevent-dev
+      - libmagic-dev
+      - libsodium-dev
+      - libuv-dev
+      - linux-headers
+      - luajit-dev
+      - ninja
+      - openssl-dev
+      - pcre2-dev
+      - perl
+      - pkgconf
+      - python-3
+      - ragel
+      - sqlite-dev
+      - wolfi-base
+      - zlib-dev
```

```suggestion
      - build-base
      - busybox
      - glib-dev
      - icu-dev
      - libarchive-dev
      - libsodium-dev
      - luajit-dev
      - openssl-dev
      - pcre2-dev
      - perl
      - ragel-dev
```
Got away with just this